dev/release/src/google-calendar.ts

```diff
@@ -85,6 +87,7 @@ export async function ensureEvent(
             end: { date: endDate, dateTime: endDateTime },
             description,
             summary: title,
+            transparency: transparency,
```

Comment:
```suggestion
            transparency,
```
you can do this in TS/JS :)